### PR TITLE
there is no "link above"

### PR DESCRIPTION
**Pull request by Brasco**
The "link above" was an article referred in the owasp cheat sheet

Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR closes #< insert number here >.

### CODE REVIEW

File: Document/0x04e-Testing-Authentication-and-Session-Management.md [hunk 1]
```diff
@@ -380,7 +380,7 @@ Verify that the implementation adheres to JWT [best practices](https://stormpath
 
 #### Enforcing the Hashing Algorithm
 
-An attacker executes this by altering the token and, using the 'none' keyword, changing the signing algorithm to indicate that the integrity of the token has already been verified. As explained at the link above, some libraries treated tokens signed with the none algorithm as if they were valid tokens with verified signatures, so the application will trust altered token claims.
+An attacker executes this by altering the token and, using the 'none' keyword, changing the signing algorithm to indicate that the integrity of the token has already been verified. Some libraries treated tokens signed with the none algorithm as if they were valid tokens with verified signatures, so the application will trust altered token claims.
```
Comment (cpholguera) [inline Document/0x04e-Testing-Authentication-and-Session-Management.md:383]: Hi @Brasco, thanks for the PR. I'd say the author wanted to refer to this link, which is indeed _above_:

https://stormpath.com/blog/jwt-the-right-way

There you can find this paragraph:

> Always verify the signature before you trust any information in the JWT. This should be a given, but we have recently seen security vulnerabilities in other company’s JWT frameworks. One gotcha that we have seen recently is around the JWT spec that allows you to set signature algorithm to ‘none’. This should be ignored if you expect the JWT to be signed. Put another way, if you are passing a secret signing key to the method that verifies the signature and the signature algorithm is set to ‘none’, it should fail verification.

I suggest the following change. What do you think? If you agree you can simply click on "Commit suggestion", else let me know your thoughts.

```suggestion
An attacker executes this by altering the token and, using the 'none' keyword, changing the signing algorithm to indicate that the integrity of the token has already been verified. [Some libraries](https://stormpath.com/blog/jwt-the-right-way) might treat tokens signed with the 'none' algorithm as if they were valid tokens with verified signatures, so the application will trust altered token claims.
```

**Review by cpholguera — CHANGES_REQUESTED** [review]
Thanks @Brasco, please take a look at my comment.

File: Document/0x04e-Testing-Authentication-and-Session-Management.md (hunk 1 above)
Comment (Brasco) [reply Document/0x04e-Testing-Authentication-and-Session-Management.md:383]: good! thank you!

**Review by cpholguera — APPROVED** [review]
Thanks @Brasco!

**Pull request merged**